### PR TITLE
Accept untrusted GnuPG keys for Maintanece Requests jobs

### PR DESCRIPTION
**Pull request by mimi1vx**
https://openqa.suse.de/tests/5690302

### CODE REVIEW

File: lib/qam.pm [hunk 1]
```diff
@@ -127,9 +127,15 @@ sub remove_test_repositories {
 
 sub advance_installer_window {
     my ($screenName) = @_;
+    my $build = get_var('BUILD');
 
     send_key $cmd{next};
     die 'Unable to create repository' if check_screen('unable-to-create-repo', 5);
+    if ($build =~ m/^MR:/) {
+        if (check_screen("gpg-key-check", 30)) {
```
Comment (dzedro) [inline lib/qam.pm:135]: 30 seconds is pretty much

Comment (dzedro) [reply lib/qam.pm:135]: Please use existing tag ` import-untrusted-gpg-key` https://openqa.suse.de/tests/5671169#step/scc_registration/23

Comment (mimi1vx) [reply lib/qam.pm:135]: existing tag looks like it was created with **exact** keys  in mind, unfortunately for MR is key changing for every repo

Comment (dzedro) [reply lib/qam.pm:135]: no, exact keys have exact tag https://openqa.suse.de/tests/5671169/modules/scc_registration/steps/23/edit
`import-untrusted-gpg-key` is "universal"
Known keys have tag `import-known-untrusted-gpg-key` https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L207

Comment (mimi1vx) [reply lib/qam.pm:135]: ahh, thx

**Review by dzedro — APPROVED** [review]

**Pull request merged**